backend/src/graphql/resolver/AdminResolver.test.ts

```diff
@@ -34,6 +38,7 @@ import { sendAccountActivationEmail } from '@/mailer/sendAccountActivationEmail'
 import Decimal from 'decimal.js-light'
 import { AdminPendingCreation } from '@entity/AdminPendingCreation'
 import { Transaction as DbTransaction } from '@entity/Transaction'
+import { ContributionLink as DbContributionLink } from '@entity/ContributionLink'
```

Comment:
the entity should have quite the same name as the dbtable, because the entity mirrors the database. The TypeORM-Model will then distigue between a single dataobject or container/list/array with several dataobjects.
examples:
- entity: User**s** <-> table: user**s** <-> typeORM(s): User
- entity: Transaction**s** <-> table: transaction**s** <-> typeORM(s): Transaction
- entity: TransactionLink**s** <-> table: transaction_link**s** <-> typeORM(s): TransactionLink / TransactionLinkList
- entity: ContributionLink**s** <->table: contribution_link**s** <-> typeORM(s): ContributionLink / ContributionLinkList

so please rename the entity ContributionLink to ContributionLink**s**